.github/workflows/ci.yml

```diff
@@ -70,17 +70,17 @@ jobs:
           command: fmt
           args: --all -- --check
 
- clippy:
```

Comment:
@mrwwheat7 you had committed bad ci.yml, so the whole github action failed to run.
please pay attention to the ci file, it's important to protect our code.